src/napari_deeplabcut/misc.py

```diff
@@ -27,6 +27,34 @@ def encode_categories(
     return inds
 
 
+def merge_multiple_scorers(
+    df: pd.DataFrame,
+) -> pd.DataFrame:
+    n_frames = df.shape[0]
+    header = DLCHeader(df.columns)
+    n_scorers = len(header._get_unique("scorer"))
+    if n_scorers == 1:
+        return df
+
+    n_bodyparts = len(header.bodyparts)
+    if "likelihood" in header.coords:
+        # Merge annotations from multiple scorers to keep
+        # detections with highest confidence
+        data = df.to_numpy().reshape((n_frames, n_scorers, n_bodyparts, -1))
```

Comment:
This fails when there are unique bodyparts, as this will not be the correct shape. I'll share some data where it fails

Reply:
Fixed!